src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSScoreInfo.java

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.hellbender.tools.spark.pathseq;
+
+/**
+ * Scores assigned to a taxonomic node, used in ClassifyReads
```

Comment:
Added some clarification to what this class is for. I have added a comment referencing the ClassifyReads tool documentation (upcoming PR), since the specifics of the scoring are actually calculated there.